anchore_engine/db/entities/policy_engine.py

```diff
@@ -163,7 +163,7 @@ class GrypeDBFeedMetadata(Base):
         nullable=False,
     )
     synced_at = Column(DateTime, nullable=True)
-    groups = Column(JSONB)
+    groups = Column(JSONB, default=[])
```

Comment:
Since the expectation is that groups will be an array and will need to be iterated over, I figured it would be easier to default it on the db level so that potential loops dont error out if its `None`.  Thought the db level might be a safer and more reliable location in case there are any errors that occur in processes and it centralizes that attribute being set to one location. Want me to do it at object creation?